workspaces/src/network/account.rs

```diff
@@ -169,14 +169,24 @@ impl<'a, T: Network> CallBuilder<'a, T> {
             signer,
             contract_id,
             function,
-            args: None,
+            args: serde_json::json!({}).to_string().into_bytes(),
             deposit: None,
             gas: None,
         }
     }
 
     pub fn with_args(mut self, args: Vec<u8>) -> Self {
-        self.args = Some(args);
+        self.args = args;
+        self
+    }
+
+    pub fn with_args_json(mut self, args: serde_json::Value) -> Self {
```

Comment:
prob more convenient to just have this generic on serde serialize, to avoid a user to convert to a serde_json::Value and avoid exposing this detail to the user?

Reply:
sounds good to me, and added